### PR TITLE
Remove (most) references to Labour and Liverpool

### PR DESCRIPTION
**Pull request by blongden73**
Removes the references in the html and css to labour and liverpool. The series is now over and is being changed to become a broader article type - "Explore".

For now we're not having a new tone. So it will still be tied to tags - but there will be a couple of tags running at the same time - hence the need to make the template look for multiple tags. Eventually if this proves successful there will need to be a more permanent solution. 

@SiAdcock @NataliaLKB - Could you take a look, that would be great!

### CODE REVIEW

File: common/app/views/fragments/labourLiverpoolSeries.scala.html [hunk 1]
```diff
@@ -9,16 +9,16 @@
 @defining((
```
Comment (SiAdcock) [inline common/app/views/fragments/labourLiverpoolSeries.scala.html]: Should we rename this file to `exploreSeries.scala.html`? Or is this template still going to be specific to Labour & Liverpool?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
 @import '../_vars';
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Should we rename this file to `article-explore.scss`?

File: common/app/views/fragments/labourLiverpoolSeries.scala.html (hunk 1 above)
Comment (blongden73) [inline common/app/views/fragments/labourLiverpoolSeries.scala.html]: Yes, i shall do that good call!

File: static/src/stylesheets/inline/article-labour-liverpool.scss (hunk 2 above)
Comment (blongden73) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Yeah.

File: article/app/views/fragments/articleBodyHeroic.scala.html [hunk 3]
```diff
@@ -28,9 +28,9 @@
                 <div class="gs-container">
                     <div class="content__main-column content__main-column--article js-content-main-column">
                         <div class="content__article-body from-content-api js-article__body" itemprop="articleBody">
-                            @if(article.content.tags.isLabourLiverpoolSeries) {
-                                <img class="labour-liverpool-cutout hide-on-mobile" src="@ImgSrc("https://static.guim.co.uk/sys-images/Guardian/Pix/contributor/2015/8/18/1439913873894/Ewen-MacAskill-R.png", Item140)">
-                                @fragments.inlineSvg("quote", "icon", List("labour-liverpool-quotes"))
+                            @if(article.content.tags.isExploreSeries) {
```
Comment (NataliaLKB) [inline article/app/views/fragments/articleBodyHeroic.scala.html]: is the `isExploreSeries` different from the "Heroic template"? If not, this file should be renamed and there shouldn't be a need for this if statement

File: common/app/model/meta.scala [hunk 4]
```diff
@@ -648,6 +648,7 @@ final case class Tags(
 
   // Specific Series
   lazy val isLabourLiverpoolSeries = tags.exists(t => t.id == "membership/series/labour-liverpool")
```
Comment (NataliaLKB) [inline common/app/model/meta.scala]: Is this still being used? If not, it should be removed. If it is, then it should still be part of the isHeroic definition in common/app/model/content.scala

Comment (blongden73) [inline common/app/model/meta.scala]: Done - changed to private

File: article/app/views/fragments/articleBodyHeroic.scala.html (hunk 3 above)
Comment (blongden73) [inline article/app/views/fragments/articleBodyHeroic.scala.html]: Names changed

File: article/app/views/fragments/articleBodyExplore.scala.html [hunk 5]
```diff
@@ -42,11 +40,6 @@
                 </div>
             </div>
         </article>
-
-        @if(article.content.tags.isLabourLiverpoolSeries) {
-            @fragments.labourLiverpoolContact()
-        }
-
```
Comment (SiAdcock) [inline article/app/views/fragments/articleBodyExplore.scala.html]: 🎉

**Comment by SiAdcock** [comment]
Can we delete the [labourLiverpoolContact.scala.html](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/labourLiverpoolContact.scala.html) template?

**Comment by SiAdcock** [comment]
There's a reference to `labour-liverpool-headline` in [`media/main.js`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/media/main.js#L175). Does this need to be updated to `explore-series-headline`?

**Comment by SiAdcock** [comment]
👍  looks good, dude, awesome work! 

As discussed, presentation issues around the sizing of the header image are not related to this change and can be addressed in a separate PR.

**Pull request merged**